Iceberg-Pharo7.package/FTTableMorph.extension/instance/deselectAll.st

```diff
@@ -0,0 +1,3 @@
+*Iceberg-Pharo7
+deselectAll 
```

Comment:
Why it is an iceberg extension? It is usefull method and deserve to be part of fast table